### PR TITLE
Update debian example configuration file

### PR DESCRIPTION
**Pull request by shivamMg**
Correct document comments.

### CODE REVIEW

File: debian/syslog-ng.conf.example [hunk 1]
```diff
@@ -38,19 +38,19 @@ destination syslog { file("/var/log/syslog.log"); };
...
 
-# Logging for INN news system
+# Logging for INN news system.
 #
 destination newscrit { file("/var/log/news/news.crit"); };
 destination newserr { file("/var/log/news/news.err"); };
 destination newsnotice { file("/var/log/news/news.notice"); };
 
-# Some `catch-all' logfiles.
+# Some `catch-all` logfiles.
```
Comment (bkil-syslogng) [inline debian/syslog-ng.conf.example]: This is a text file and not markdown syntax. In our documentation, we use the convention shown above, so I would leave this one.

**Comment by bkil-syslogng** [comment]
Thank you for your improvements. Please address the issue that I have noted above, but otherwise I like and welcome the changes.

**Comment by shivamMg** [comment]
@bkil-syslogng I have squashed the change in the previous commit.

**Comment by dnsjts** [comment]
:+1:

**Pull request merged**